ondewo/nlu/agent_pb2_grpc.py

```diff
@@ -737,476 +738,476 @@ class Agents(object):
 
     @staticmethod
     def CreateAgent(request,
```

Comment:
I guess would be better if you make enter before `request` and after `metadata=None` keeping the paragraphs as it was before. Just a suggestion